backend/app/management/commands/syncdb.py

```diff
@@ -113,16 +120,49 @@ def handle(self, *args, **options):
 
             for row in values_as_a_dict:
                 print(row)
+                # Filter column headers for model fields
+                model_fields = MODEL_NAME_TO_MODEL[model_name]._meta.get_fields()
+                model_field_names = [field.name for field in model_fields]
+                model_kwargs = {}
+                for header in row.keys():
+                    if header in model_field_names or header == 'map_square_number':
+                        # Check if value in column is a number
+                        value = row[header]
+                        if header in ['number', 'map_square_number', 'photographer']:
+                            try:
+                                value = int(value)
+                                if header == 'map_square_number':
+                                    header = 'map_square'
+                            except ValueError:
+                                continue
+                        # Evaluate value as a boolean
+                        elif header == 'contains_sticker':
+                            if value.lower() == 'yes':
```

Comment:
This is cool, but I think we should just agree on a way of coding this data in the sheet, and handle similarly to the above ValueError case in case of incorrect coding.